Tasks/DownloadSecureFile/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -4,5 +4,7 @@
   "loc.description": "Download a secure file to a temporary location on the build or release agent",
   "loc.instanceNameFormat": "Download secure file",
   "loc.input.label.secureFile": "Secure File",
-  "loc.input.help.secureFile": "Select the secure file to download to a temporary location on the agent. The file will be cleaned up after the build or release."
+  "loc.input.help.secureFile": "Select the secure file to download to a temporary location on the agent. The file will be cleaned up after the build or release.",
```

Comment:
None of the resources.resjson/* files need to be updated. These will be localized by a vendor team later.